### ISSUE TITLE
Ternary syntax

### ISSUE THREAD

**Issue by kiliandys** [Quick-Turn-Studio/CLionSupportForQt#11]
The following code generate an error `<ternary value> expected, got '['`

```
readonly property var methods: experimentalFeatures ? [
	    { uuid: method1, title: qsTr('method1') },
	    { uuid: method2, title: qsTr('method2') },
	    { uuid: method3, title: qsTr('method3') },
    ] : [
        { uuid: method1, title: qsTr('method1') },
        { uuid: method2, title: qsTr('method2') },
    ]
```

**Comment by grabusr** [comment]
Hello Kilian

Thank you for reporting bug. We will fix it soon.

**Comment by podlaszczyk** [comment]
Fixed in 0.9.7
